### ISSUE TITLE
Keyboard shortcuts reference not accessible to screen readers

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6118]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.795
    OS Version      : Any
    R Version       : Any

### Steps to reproduce the problem
With a screen reader on, do Help / Keyboard Shortcuts Help to display keyboard shortcuts reference, then try to read it via screen reader.

### Describe the problem in detail
Any attempt to use the screen reader immediately dismisses the popup.

### Describe the behavior you expected
It is important for a screen reader user to become familiar with RStudio shortcuts to use the product efficiently. Though many can be found by browsing the main menu, many of them are only associated with toobar buttons in the various panes, making them even harder to discover. Having a single-place to browse shortcuts would be very helpful.

When screen-reader mode is enabled, bypass the popup-up and directly load the static HTML keyboard shortcut page normally reached via the (also not accessible) link "See all Shortcuts..." in upper-right of popup. This will allow browsing it via normal HTML screen reader commands.

See also #6117 for some issues with that static page.

**Comment by ronblum** [comment]
Confirmed in RStudio Server and Desktop on Mac and Windows. Works as expected with screen reader support on and off.
